pkg/workceptor/remote_work.go

```diff
@@ -113,7 +111,7 @@ func (rw *remoteUnit) getConnection(mw *utils.JobContext) (net.Conn, *bufio.Read
 func (rw *remoteUnit) connectAndRun(ctx context.Context, action actionFunc) error {
 	conn, reader, err := rw.connectToRemote(ctx)
 	if err != nil {
-		return utils.WrapErrorWithKind(err, "connection")
+		return utils.WrapErrorWithKind(err, "connection first attempt")
```

Comment:
What's going on here?

Reply:
accidentally left this in, I'll fix in my next commit